content/md/en/docs/main-docs/build/build-process.md

```diff
@@ -1,118 +1,155 @@
-```
+1. Display an inverted dependency graph by running the following command:
 
-But how is the Wasm runtime created?
-What really happens when `cargo build --release` is executed in the directory of a Substrate node project?
+    ```bash
+    cargo tree -i node-template-runtime
+    ```
 
-At a very high level, this command builds the project in it's local directory &mdash; including both native and Wasm binaries &mdash; with optimized artifacts, meaning that the output executable goes through some post-processing.
-These artifacts then result in the final executable program that enables launching a chain with the following command:
-`./target/release/node-template --dev`.
+    The command displays output similar to the following:
 
-During the build process, the Wasm runtime binary goes through 3 different stages, each containing the various steps.
-In the first stage of the build cycle, the initial Wasm runtime is built and embedded into the client.
-Once the node is compiled, the compressed Wasm binary is placed on-chain at the [`:code`](https://docs.substrate.io/rustdocs/latest/sp_storage/well_known_keys/constant.CODE.html) storage key and executed by the client.
+    ```bash
+    node-template-runtime devhub/latest
+    └── node-template devhub/latest
+    ```
 
-The following steps describes the entire build process:
+## Wasm runtime
 
-**A. Create the initial runtime Wasm binary**
+You probably already know that you can compile a Substrate node by running the `cargo build --release` command in the root directory for a Substrate node project.
+This command builds both the Rust and Wasm binaries for the project in the local directory and produces an **optimized** executable artifact.
+Producing the optimized executable artifact includes some post-compilation processing to result in a program that enables you to launch a blockchain that includes both runtime binaries.
 
-- Cargo builds the entire graph of dependencies in all project TOML files.
-- The runtime's `build.rs` module uses the `substrate-wasm-builder` crate.
-- This `build.rs` module executes and compiles the runtime into a Wasm binary, creating `wasm_binary.rs`, i.e. the initial wasm (largest size). 
+As part of the optimization process, the Wasm runtime binary is compiled, compressed, and placed on-chain through a series of internal steps.
+To give you a better understanding of the process, the following diagram summarizes the steps.
 
-**B. Post-processing**
+![WebAssembly compiled and compressed before placed on-chain](/media/images/docs/main-docs/node-executable.png)
 
-- Then the `substrate-wasm-builder` wasm linker invokes the optimizations to create a Compact wasm binary.
-- It optimizes some instruction sequences and removes any unnecessary sections, such as the ones for debugging, using a tool called [wasm-opt](https://www.npmjs.com/package/wasm-opt).
-- The runtime crate is a dependency of the node.
+The following sections describe the build process in more detail.
```

Comment:
Not mentioning the Wasm builder seems like we are missing some key information on this whole page: https://github.com/paritytech/substrate/tree/master/utils/wasm-builder